index.js

```diff
@@ -203,31 +284,31 @@ smartapp
     .subscribedEventHandler('myContactEventHandler', (context, deviceEvent) => {
         const signal = deviceEvent.value;
         console.log(deviceEvent.value);
-        const LocationID = context.event.eventData.installedApp.config.LocationID[0].stringConfig.value;
-        const DeviceID = context.event.eventData.installedApp.config.DeviceID[0].stringConfig.value;
+        const LocationID = context.event.eventData.locedApp.config.LocationID[0].stringConfig.value;
+        const DeviceID = context.event.eventData.locedApp.config.DeviceID[0].stringConfig.value;
```

Comment:
What are these changes in the SmartThings handler about?

Reply:
I just fixed them, I was being lazy and trying to replace bits of copied in code 😓

Reply:
[install]ation to [loc]ation 🤐